### ISSUE TITLE
why usebias = nn.InstanceNorm2D

### ISSUE THREAD

**Issue by MakeDirtyCode** [junyanz/pytorch-CycleGAN-and-pix2pix#431]
https://github.com/junyanz/pytorch-CycleGAN-and-pix2pix/blob/1fa94d7074fcc94ac108386799738e46f19317c0/models/networks.py#L150

Why use bias nn.InstanceNorm2d here?
What is the difference between using True?

**Comment by junyanz** [comment]
`InstanceNorm` doesn't have affine parameters (by default). So we add a bias term. 
`BatchNorm` has affine parameters so there is no need for the bias term.

**Comment by MakeDirtyCode** [comment]
Aha. Then it was bias = False for batch normalization and bias = True for instance normalization.

**Comment by junyanz** [comment]
Yes.

**Comment by MakeDirtyCode** [comment]
Great. Thanks

**Comment by TJJTJJTJJ** [comment]
Even if InstanceNorm doesn't have affine parameters, under the calculation x-E(x), it also remove the influence of use_bias. Maybe somewhere I'm wrong, please check me out if you're free.

**Comment by junyanz** [comment]
not sure. The cycle consistency loss might affect the use_bias for the G's layer. I don't think we use the bias for the G's last layer anyway. use_bias could affect both G and D.
